### PR TITLE
feat: update tab styling

### PR DESCRIPTION
**Pull request by benfurber**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

Updates the tab design in line with the new impact data feature.

## Screenshots/Videos

Screenshot 2023-11-28 at 11 08 51
Screenshot 2023-11-28 at 11 09 09
Screenshot 2023-11-28 at 11 08 20
Screenshot 2023-11-28 at 11 08 00

---

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of our monthly maintainers call (first Monday of the month)

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on Discord in the [Community Platform `development` channel](https://discord.com/channels/586676777334865928/938781727017558018).

### CODE REVIEW

**Comment by benfurber** [comment]
FYI. Because of the beta testing wrappers, it technically looks like this atm: 
Screenshot 2023-11-28 at 11 13 08

**Comment by davehakkens** [comment]
Looks nice! 👌

File: src/pages/User/impact/ImpactField.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const ImpactField = ({ field }: Props) => {
   const { label, prefix, suffix, value } = field
 
   const sx = {
-    backgroundColor: 'white',
+    backgroundColor: 'background',
```
Comment (evakill) [inline src/pages/User/impact/ImpactField.tsx:16]: is this a real thing?! what does it do?

Comment (benfurber) [reply src/pages/User/impact/ImpactField.tsx:16]: @evakill Sorry, I'd missed this. Do you mean what does 'background' link to? It's one of the ~colours~ colors in the base theme `packages/themes/src/common/commonStyles.ts`

Comment (evakill) [reply src/pages/User/impact/ImpactField.tsx:16]: ooooh got it

**Pull request merged**